e2e/fixtures.ts

```diff
@@ -0,0 +1,13 @@
+import { Locator, test as base } from '@playwright/test'
```

Comment:
Open to suggestions about what to name this file (`fixtures.ts` isn't that accurate, but something like `test.ts` might be confusing).

Reply:
hmm difficult one I would have used test-util.ts but that is already taken. I agree test.ts is confusing